.github/workflows/ci.yml

```diff
@@ -121,6 +121,8 @@ jobs:
 
   bundle-analyzer:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
Can we add the permissions to just what needs it? I believe the `transferwise/actions-next-bundle-analyzer@v2` step is what creates the comment. Can we make only that step have the write permission?

Reply:
That's a good idea, but according to [the docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#overview), it seems that you can set permissions for an entire workflow or for a specific job, but not for a specific step in a job

> You can use permissions either as a top-level key, to apply to all jobs in the workflow, or within specific jobs. When you add the permissions key within a specific job, all actions and run commands within that job that use the GITHUB_TOKEN gain the access rights you specify.